Client/Frontend/Toolbar+URLBar/URLBarView.swift

```diff
@@ -236,7 +236,13 @@ class URLBarView: UIView, URLBarViewProtocol, AlphaDimmable, TopBottomInterchang
         }
 
         set(newURL) {
-            locationView.url = newURL
+            // Ecosia: Update URL accordingly
+            var updatedUrl = newURL
+            if updatedUrl?.isEcosiaSearchQuery() == true {
+                updatedUrl = newURL?.ecosified(isIncognitoEnabled: isPrivate)
+            }
+            locationView.url = updatedUrl
+            // locationView.url = newURL
```

Comment:
Can you just move the commented code next to the comment? I believe that is the patter we use

```suggestion
            // Ecosia: Update URL accordingly
            // locationView.url = newURL
            var updatedUrl = newURL
            if updatedUrl?.isEcosiaSearchQuery() == true {
                updatedUrl = newURL?.ecosified(isIncognitoEnabled: isPrivate)
            }
            locationView.url = updatedUrl
```